app/controllers/account_requests_controller.rb

```diff
@@ -51,6 +53,11 @@ def set_account_request_from_token
     @account_request
   end
 
+  # Pass parameter through to 'render :new' action to expand account_request form upon form or captcha error reload
+  def bank_selected
```

Comment:
This seems like a bit of a redundant method? The calling code can just set `@bank_selected` to true.

Reply:
🤦‍♂️  You are absolutely right, haha. Thank you for pointing this out! 

I'm seeing now that I have absolutely overcomplicated this task, and learning that I need to focus more on keeping it simple and should dig further into resources. 😅 

I have made the following updates:
- Removed the erroneous `@bank_selected` from the #new controller action
- Removed the redundant private `bank_selected` method
- Updated the #create controller action to hold this value by setting the variable simply as `@bank_selected =  true`
- Removed the unnecessary `:bank_selected` hidden field from the form (and all script related to it)
- I also noticed there was a check that did not pass due to linting errors and have run Rubocop on the relevant pages I worked on.


Please let me know if you think anything else should be adjusted. And seriously, thank you for your patience and guidance. I've learned a lot more about how re-rendering operates within Rails - appreciate you!

Reply:
My pleasure! Glad it was a good experience for you. :smile: